OTAnalytics/plugin_ui/customtkinter_gui/frame_bbox_offset.py

```diff
@@ -68,3 +72,18 @@ def get_relative_offset_coordintes(self) -> dict:
         x = round(self.slider_x.get(), 2)
         y = round(self.slider_y.get(), 2)
         return {geometry.X: x, geometry.Y: y}
+
+    def _on_slider_change(self, value: Any) -> None:
+        x, y = self._get_slider_values()
+        self.label_x_value.configure(text=round(x, 1))
+        self.label_y_value.configure(text=round(y, 1))
+
+    def _on_slider_release(self, value: Any) -> None:
+        x, y = self._get_slider_values()
+        if self._notify_change is not None:
+            self._notify_change(round(x, 2), round(y, 2))
```

Comment:
Why do we round to a different number of decimal digits?

Reply:
I changed ot to the same everywhere and also extracted out a constant to the top of the module. It depends on the slider resolution, which I also extracted out to a constant.